src/webapp/pages/settings/SettingsPage.tsx

```diff
@@ -94,18 +108,18 @@ export const SettingsPage: React.FC = () => {
     }, [danglingDocuments, loading, snackbar, usecases]);
 
     const refreshModules = useCallback(async () => {
-        usecases.instance.listDanglingDocuments().then(setDanglingDocuments);
         await reload();
-    }, [reload, usecases]);
+    }, [reload]);
 
     const openAddModule = useCallback(() => {
         setAppState({ type: "CREATE_MODULE" });
     }, [setAppState]);
 
     const toggleShowAllModules = useCallback(async () => {
-        await usecases.config.setShowAllModules(!showAllModules);
-        await reload();
-    }, [showAllModules, reload, usecases]);
+        save({
```

Comment:
`await`

Reply:
Well, being a useCallback and the result not being used, it's indifferent if async or await are used. This should be a useEffect, but let's keep a useCallback for simplicity.